src/Future.ts

```diff
@@ -19,6 +21,60 @@ declare module 'fp-ts/lib/HKT' {
  */
 export const URI = 'Fluture/Future'
 
+/**
+ * @since 0.6.4
+ */
+export interface Future<E, A> extends F.FutureInstance<E, A> {}
+
+/**
+ * @since 0.6.4
+ */
+export const left: <E = never, A = never>(e: E) => Future<E, A> = F.reject
+
+/**
+ * @since 0.6.4
+ */
+export const right: <E = never, A = never>(a: A) => Future<E, A> = F.resolve
+
+/**
+ * @since 0.6.4
+ */
+export function orElse<E, A, M>(onLeft: (e: E) => Future<M, A>): (ma: Future<E, A>) => Future<M, A> {
+  return ma => F.chainRej(onLeft)(ma)
+}
+
+/**
+ * @since 0.6.4
+ */
+export function fork<E, A>(onLeft: (e: E) => void, onRight: (a: A) => void): (ma: Future<E, A>) => () => void {
```

Comment:
This is not idiomatic in `fp-ts`, a function returning `void` either is costant or performs side effects

Reply:
You're right 👍 

`fork` and `value` as well are not very useful at all in this module, I am removing both.

Thank you so much for taking the time to review these changes. I was playing around with `hyper-ts` and found my way all the way up to this module again! thought that would be nice to make it consistent with `TaskEither`.